core/mutator/entry/mutation.go

```diff
@@ -59,9 +58,13 @@ func NewMutation(index []byte, domainID, appID, userID string) *Mutation {
 	}
 }
 
+// CopyPrevious indicates whether to overwrite all values in the current entry with
+// the previous entry's values.
+type CopyPrevious bool
```

Comment:
I don't think there's a readability gain from having a typedef'd bool.

In SetPrevious itself, you write 'if copyPrevious { ... }'
You'd write that whether or not you define the type here.

If you want the extra type safety / documentation then making this more of an enum so that values are named as well would be more helpful.

```
type CopyPrevious int
const (
  copyPrevious = iota
  dontCopyPrevious
)
```

maybe?

Reply:
Should I use a plain `bool` then?